app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
```

Comment:
Order imports in alphabetical order to follow PEP-8

Reply:
done